packages/http-client-csharp/eng/scripts/Test-Packages.ps1

```diff
@@ -24,7 +24,7 @@ try {
             # test the generator
             Invoke-LoggedCommand "dotnet test ./generator" -GroupOutput
 
-            Invoke-LoggedCommand "./eng/scripts/Test-CadlRanch.ps1" -GroupOutput
```

Comment:
Is Test-CadlRanch still used?

Reply:
used locally by us since its much faster in most local scenarios.